web/src/settings_profile_fields.ts

```diff
@@ -732,8 +798,8 @@ export function build_page() {
     $("#admin_profile_fields_table").on("click", ".open-edit-form-modal", open_edit_form_modal);
     $("#admin_profile_fields_table").on(
         "click",
-        ".display_in_profile_summary",
+        "input.display_in_profile_summary",
         toggle_display_in_profile_summary_profile_field,
     );
-    $("#admin_profile_fields_table").on("click", ".required-field-toggle", toggle_required);
+    $("#admin_profile_fields_table").on("click", "required-field-toggle", toggle_required);
```

Comment:
This results in a bug where the `required` field is not updated on clicking the checkbox directly from the table.

Reply:
Oops, yes should I make a quick PR?

Reply:
Sincere apologies for the oversight, I've made the PR to fix this #30154